utils.py

```diff
@@ -8,6 +10,17 @@ def get_date(forex):
     return forex.date
 
 
+def round_decimal(number, precision: str = "0.01"):
```

Comment:
I'd go with the usual convention and set the precision to zero (or omit the default value)

Reply:
But it technically is the default here in terms of representing values and prices denoted in some currency, right?